src/journal/journald-server.c

```
@@ -676,6 +676,14 @@ static int server_archive_offline_user_journals(Server *s) {
 
                 TAKE_FD(fd); /* Donated to journal_file_open() */
 
+#if HAVE_GCRYPT
+                /* Write the final tag */
+                if (JOURNAL_HEADER_SEALED(f->header) && journal_file_writable(f)) {
+                        r = journal_file_append_tag(f);
+                        if (r < 0)
+                                log_error_errno(r, "Failed to append tag when closing journal: %m");
```

The failure will be ignored. So, please downgrade the log level, at most LOG_WARNING. Also, please append `", ignoring"` to the log message.